commercial/src/dig_components/create_company.js

```diff
@@ -6,45 +6,46 @@ import NumberFormat from 'react-number-format';
 
 function CreateCompany(props) {
 	// console.log('Create Company Refresh')
-	let history = useNavigate();
+	const history = useNavigate();
```

Comment:
Naming this `history` makes it seem like we are only going backwards in history when using this. I would rename to `navigate` or something